docs/core/config/auth/databases/ldap.md

```diff
@@ -220,12 +235,12 @@ reconnecting to the "primary" server.
 
 You can enable TLS in two alternative ways:
 
-* Connect to ldaps port (636) by using "ldaps" protocol, e.g. `uris =
+* Connect to ldaps port (636) by using "ldaps" protocol, e.g. `ldap_uris =
   ldaps://secure.domain.org`
-* Connect to ldap port (389) and use STARTTLS command. Use `tls=yes` to
+* Connect to ldap port (389) and use STARTTLS command. Use `ssl=yes` to
   enable this.
```

Comment:
ldap_ssl=yes .. hmm. This probably should be renamed to ldap_starttls..

Reply:
code has already been merged ... what to do with this one ?